commons/tool/src/java/org/sakaiproject/commons/tool/entityprovider/CommonsEntityProvider.java

```diff
@@ -500,4 +517,32 @@ private String getCheckedUser() throws EntityException {
     private String escape(String unescaped) {
         return StringEscapeUtils.escapeJava(unescaped);
     }
+
+    private void sendPriorityEmail(String siteId, String postContent){
+        try{
+            Set<String> siteUserIds = siteService.getSite(siteId).getUsers();
+            Set<User> siteUsers = new HashSet<User>();  //will be list of users to email
+            for(String id: siteUserIds){
+                try{
+                    siteUsers.add(userDirectoryService.getUser(id));
+                } catch (UserNotDefinedException u){
+                    log.error("No user found with ID " + id + " while sending Commons Priority email.");
```

Comment:
Newer log style without string concatenation:

```suggestion
                    log.error("No user found with ID {} while sending Commons Priority email.", id);
```

Reply:
Updated it as requested!